common/djangoapps/track/event_transaction_utils.py

```diff
@@ -0,0 +1,48 @@
+"""
+Helper functions to access and update the id and type
+used in event tracking.
+"""
+from uuid import uuid4, UUID
+from request_cache import get_cache
+
+
+def get_event_transaction_id():
+    """
+    Retrieves the current event transaction id from the request
+    cache.
+    """
+    return get_cache('event_transaction').get('event_transaction_id', None)
```

Comment:
Nitty: Since the cache is now named `event_transaction`, you don't need to include it in the cached data `dict`.  So, the cached data dict key names can just be "id" and "type".